javascript/ql/test/query-tests/Security/CWE-601/ClientSideUrlRedirect/react.js

```diff
@@ -15,3 +15,23 @@ class Application extends React.Component {
 };
 
 export default Application
+
+import Link from 'next/link'
+export function NextLink() {
+    return <Link href={document.location.hash}><a>this page!</a></Link>;
```

Comment:
Did you test this?

Reply:
Yes.  
React gives a warning that in the future `javascript:` links will be disabled, but for now it works.